pkgs/development/python-modules/pysqlite3/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, python
+, pytestCheckHook
+, sqlite
+}:
+
+buildPythonPackage rec {
+  pname = "pysqlite3";
+  version = "0.5.2";
+
+  src = fetchPypi {
```

Comment:
Consider fetching using GitHub instead.

Reply:
Thank you for the suggestion. Why should I use `fetchFromGitHub` instead of `getchPypi` for this kind of packages?

Reply:
From my experience, it is common for pypi uploads to miss sources for tests and documentation, so I default to GitHub tarballs.

And git is better idea of `source` than `derivative tarball on pypi`, imho.

Reply:
@KAction fixed as requested.